GenerateEverything.hs

```diff
@@ -0,0 +1,132 @@
+{-# LANGUAGE PatternGuards #-}
+
+import qualified Data.List as List
+import Control.Applicative
+import System.Environment
+import System.IO
+import System.Exit
+import System.FilePath
+import System.FilePath.Find
+
+headerFile     = "Header"
+allOutputFile  = "Everything"
+srcDirs        = ["Cubical/HITs", "Cubical/Basics"]
+
+excludedModules :: [FilePath]
```

Comment:
I guess the modules listed here won't be listed in the Everything file?

Note that HInt has moved to Experiments now, so I don't think we have to list it. I would also not list HitInt and SmashProduct.

Reply:
> Why SmashProduct

I see now. Updated.